src/main/java/com/zendesk/maxwell/bootstrap/BootstrapController.java

```diff
@@ -83,6 +84,21 @@ public synchronized void setCurrentSchemaID(long schemaID) {
 		this.currentSchemaID = schemaID;
 	}
 
+	public boolean hasIncompleteTasks() {
+		try ( Connection cx = maxwellConnectionPool.getConnection() ) {
+			PreparedStatement s = cx.prepareStatement("select count(id) from bootstrap where is_complete = 0 and client_id = ?");
+			s.setString(1, this.clientID);
+
+			ResultSet rs = s.executeQuery();
+
+			while (rs.next()) {
+				return rs.getInt(1) > 0;
+			}
+		} catch (Exception ex){}
```

Comment:
Is it possible for this code to live in `src/test/java` since it is purely for test purposes?  Or if it is intended for wider use and needs to live in `src/main/java`, perhaps the details of the swallowed Exception should be logged?